src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/JsonFormat/SearchResultPackagesConverter.cs

```diff
@@ -33,11 +43,11 @@ public override void Write(Utf8JsonWriter writer, IPackageSearchMetadata value,
 
             if (_exactMatch)
             {
-                writer.WriteString(JsonProperties.Version, value.Identity.Version.ToNormalizedString());
+                WriteStringIfNotNullOrWhiteSpace(writer, JsonProperties.Version, value.Identity.Version?.ToNormalizedString());
```

Comment:
nit: Normalizing the version seems like it would be part of the helper method instead of done beforehand.